test/colony-recovery.js

```diff
@@ -133,7 +139,7 @@ contract("Colony Recovery", accounts => {
       await colony.setStorageSlotRecovery(5, "0xdeadbeef");
 
       const unprotected = await web3GetStorageAt(colony.address, 5);
-      assert.equal(unprotected.toString(), `0xdeadbeef${"0".repeat(56)}`);
+      expect(unprotected).to.eq.BN(`0xdeadbeef${"0".repeat(56)}`);
```

Comment:
We ought to find a better way to pad variables to correct length.. But I guess it's not the purpose of this PR

Reply:
Fair fair. I was able to get rid of `parseInt` though :)